plugins/modules/zos_fetch.py

```diff
@@ -170,66 +171,66 @@
 
 RETURN = r"""
 file:
-    description: The source file path on remote machine
+    description: The source file path on the remote machine.
     returned: success
     type: str
     sample: SOME.DATA.SET
 dest:
-    description: The destination file path on controlling machine
+    description: The destination file path on the controlling machine.
     returned: success
     type: str
     sample: /tmp/SOME.DATA.SET
 is_binary:
-    description: Indicates which transfer mode was used to fetch the file
+    description: Indicates which transfer mode was used to fetch the file.
```

Comment:
@asifmahmud - I believe is_binary is used for both USS and Data sets, so i think its more accurate to truncate this sentence to "Indicates which transfer mode was used to fetch."

@RadhaV2020